src/ProtocolV3TestBase.sol

```diff
@@ -0,0 +1,1570 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity >=0.7.5 <0.9.0;
+
+import 'forge-std/Test.sol';
+import {
+  IAaveOracle,
+  IPool,
+  IPoolAddressesProvider,
+  IPoolDataProvider,
+  IDefaultInterestRateStrategy,
+  DataTypes,
+  IPoolConfigurator
+} from 'aave-address-book/AaveV3.sol';
```

Comment:
I think these are all provided by the `aave-v3-core` interfaces directory. Perhaps we can get rid of the address book dependency? I'm ok if you want to do this in a subsequent PR too.

Reply:
Ill add a TODO to investigate, from what I remember there was more in that repo that was needed